var/spack/repos/builtin/packages/py-setuptools-scm/package.py

```diff
@@ -22,5 +23,9 @@ class PySetuptoolsScm(PythonPackage):
 
     depends_on('python@2.7:2.8,3.4:', type=('build', 'run'))
     depends_on('python@2.7:2.8,3.5:', when='@4:', type=('build', 'run'))
+    depends_on('python@3.6:', when='@6:', type=('build', 'run'))
+
     depends_on('py-setuptools@34.4:', type=('build', 'run'))
+    depends_on('py-setuptools@45:', type=('build', 'run'), when='@6.0.1:')
```

Comment:
```suggestion
    depends_on('py-setuptools@45:', type=('build', 'run'), when='@6:')
```
Looks like 6.0.0 also has this dep.